openstudiocore/src/openstudio_lib/OSDocument.cpp

```diff
@@ -1490,7 +1490,9 @@ namespace openstudio {
   {
     bool fileSaved = false;
 
-    QString defaultDir = savePath().isEmpty() ? mainWindow()->lastPath() : QFileInfo(savePath()).path();
+    // Defaults to the current savePath is there is one (will populate the filename too)
+    // Or the lastPath (directory) if not.
+    QString defaultDir = savePath().isEmpty() ? mainWindow()->lastPath() : savePath();
```

Comment:
I thought about changing the variable name for the same reason. But considering it can either be a directory **or** a path, and Qt calls it that way I didn't. But I added the comment above to explain what's going on, and it's only used right below it, so I think it's clear enough.